composer.json

```diff
@@ -4,10 +4,10 @@
     "description": "Google Analytics integration for Neos CMS",
     "license": "GPL-3.0-or-later",
     "require": {
-        "neos/neos": "^7.0 || dev-master",
-        "neos/fusion-afx": "^7.0",
+        "neos/neos": "7 - 8 || dev-master",
+        "neos/fusion-afx": "7 - 8",
```

Comment:
Hi @bastianschwarz 

Thank you a lot for your PR!

I would prefer a slightly different syntax, as we use this on other repos.

```suggestion
        "neos/neos": "^7.0 ||  ^8.0  || dev-master"
```

And we can also remove the requirement to `neos/fusion-afx`, as this is a requirement from `neos/neos`

Reply:
Sure thing, I probably have some time over the long weekend so I will keep this in mind and probably open some more PRs for other components I use.

Will do this in bulk then